protocol/avdl/keybase1/invite_friends.avdl

```diff
@@ -0,0 +1,15 @@
+@namespace("keybase.1")
+
+protocol inviteFriends {
+  import idl "common.avdl";
+
+  record InviteCounts {
+    int inviteCount; // Number of invites in the last 24 hours
+    float percentageChange; // vs the previous 24 hours
+    boolean showFire; // Are keybase servers on fire?
+  }
+
+  void invitePeople(array<EmailAddress> emails, array<RawPhoneNumber> phones);
```

Comment:
`string emails`, parsing into individual ones won't happen in the gui

Reply:
Hmm and then for the contacts screen just concatenate all the emails? that seems sad

Reply:
We could add it as an extra param and make them both optional? I don't think we're guaranteed well formed emails from the contacts screen so there's probably some shared logic there.